src/pixiUtils.ts

```diff
@@ -5,39 +5,12 @@ const PX_PER_EM = 16;
 const PX_PER_PERCENT = 16 / 100;
 const PX_PER_PT = 1.3281472327365;
 
-export const measureFont = (context: CanvasRenderingContext2D): IFontMetrics =>
-  PIXI.TextMetrics.measureFont(context.font);
-
 export const INITIAL_FONT_PROPS: IFontMetrics = {
   ascent: 10,
   descent: 3,
   fontSize: 13,
 };
 
-// TODO: Memoize
-export const getFontPropertiesOfText = (
-  textField: PIXI.Text,
-  forceUpdate = false
-): IFontMetrics => {
-  if (forceUpdate) {
-    textField.updateText(false);
-    return measureFont(textField.context);
-  } else {
-    const props = measureFont(textField.context);
-    const fs = textField.style.fontSize ?? NaN;
-    if (
-      props.ascent === INITIAL_FONT_PROPS.ascent &&
-      props.descent === INITIAL_FONT_PROPS.descent &&
-      (isNaN(Number(fs)) || fs > INITIAL_FONT_PROPS.fontSize)
-    ) {
-      throw new Error(
-        "getFontPropertiesOfText() returned metrics associated with a Text field that has not been updated yet. Please try using the forceUpdate parameter when you call this function."
```

Comment:
Please make sure that this error is handled in your new method for calculating the metrics.